grype/db/v6/db.go

```diff
@@ -35,6 +35,14 @@ type Writer interface {
 	io.Closer
 }
 
+type Curator interface {
+	Reader() (Reader, error)
+	Status() Status
+	Delete() error
+	Update() (bool, error)
```

Comment:
What does the bool add? I'm imagining return values like:

* `(true, nil)` --> an update actually happened, with no error
* `(false, nil)` --> no update necessary
* `(true, err)` --> not sure what this would mean?
* `(false, err)` --> an error occurred and no update was made

Reply:
Notes from sync review:

> `(true, err)` --> not sure what this would mean?

The `true` is just ignored in this case. This is following the usual Go pattern of "err is last return value; if err is not nil, ignore other return values." I was just distracted from this fact because the other return value was a bool so it was possible to type up a cool table.